catroid/src/main/java/org/catrobat/catroid/cloudmessaging/NotificationSubscriber.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2017 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.cloudmessaging;
+
+import com.google.firebase.messaging.FirebaseMessaging;
+
+public final class NotificationSubscriber {
+
+	private static final String DEFAULT_TOPIC = "DEFAULT_TOPIC";
```

Comment:
Hmm, `DEFAULT_TOPIC` isn't really meaningful. What to you think about `notification_enabled`?
Then you can send it to e.g. `'notification_enabled' in topics && 'language_en' in topics`